components/airtable-demo.js

```diff
@@ -0,0 +1,14 @@
+import Airtable from 'airtable'
+
+var base = new Airtable({ apiKey: "keyQ54SEzABIZEm9F" }).base(
+  "appv6M4U4ckXcHzhU"
+);
+
+const table = base('Tasks');    
+const getRecords = async () => {
+    const records = await table.select().firstPage();
+    console.log(records);
+    return records
+}
+
+export{getRecords}
```

Comment:
Please always add a blank line in the end of each file

Reply:
okay